README.md

```diff
@@ -91,17 +91,17 @@ require("image").setup({
   integrations = {
     markdown = {
       enabled = true,
-      sizing_strategy = "auto",
-      download_remote_images = true,
       clear_in_insert_mode = false,
+      download_remote_images = true,
       only_render_image_at_cursor = false,
-      filetypes = { "markdown" }, -- markdown extensions (ie. quarto) can go here
+      filetypes = { "markdown", "vimwiki" }, -- markdown extensions (ie. quarto) can go here
```

Comment:
It might be nice to better explain what can go into a `filetypes` block. What determines if two filetypes can use the same integration? with something like quarto vs markdown, it's very intuitive b/c they have the exact same syntax for links. But vimwiki has it's own syntax.

I'm guessing the answer is just "they have the same tree sitter node structure" so maybe we should mention that in the `## Integrations` section

Reply:
Thanks!
Yep, vimwiki is treated as markdown, as it hijacks the filetype, it was introduced in https://github.com/3rd/image.nvim/pull/16
I added a note in the README about the used parsers for each integration, and a note that it only supports the Markdown flavour of VimWiki, thanks for suggesting that.